blockchain/state_manager/src/lib.rs

```diff
@@ -260,12 +261,18 @@ where
             escrow: {
                 let et = BalanceTable::from_root(self.bs.as_ref(), &market_state.escrow_table)
                     .map_err(|_x| Error::State("Failed to build Escrow Table".to_string()))?;
-                et.get(&new_addr).unwrap_or_default()
+                et.get(&new_addr)
+                    .unwrap_or_default()
+                    .to_biguint()
+                    .ok_or_else(|| Error::State("Failed to get non-negative value".to_string()))?
```

Comment:
change the type of the table to bigint, there shouldn't be any reason to convert to biguint like this (change involves switching balancetable functionality